README.md

```diff
@@ -6,6 +6,33 @@
 
 The project uses poetry for Python to create an isolated environment and manage package dependencies. To prepare your system, ensure you have an official distribution of Python version 3.8+ and install Poetry using one of the following commands (as instructed by the [poetry documentation](https://python-poetry.org/docs/#system-requirements)):
 
+For styling tailwindcss is used via their CLI tool. For this to work you must also have Node installed.
+
+### VSCode
+
+It is reccomended to use VSCode for this project as there is an included launch config and linting has so far been enforced using VSCode extensions. The reccomended linting extensions are:
+
+- Better Jinja
+- Black Formatter
+- Mypy
+- Prettier (You will also need the NPM package, installation details described in the style section)
+
+However none of these styles are enforced yet so go wild I guess.
+
+You may also want to install other extensions to make development easier such as Pylance and Tailwind CSS IntelliSense (this extension is especially helpul if you are new to tailwind).
+
+### Styles
+
+- Install the latest version of Node from: https://nodejs.org/en
+- Install the dependancies using `npm install`
+- To keep the css build updated run the cli tool with the following command (--watch will watch for file changes in the project regenerate the CSS when neccisary):
+
+```
+npx tailwindcss -i ./todo_app/tailwind.css -o ./todo_app/static/css/index.css --watch
+```
```

Comment:
**This is not important, just in case interesting**

I often feel that having a separate extra command for this kind of thing is _slightly_ annoying as a dev; this definitely isn't important but if we can bake it into the "default" run command, I always think that's nicer. I have used poetry's `scripts` section before, but it looks like it's not great at running arbitrary commands and certainly not combos ([e.g. this discussion](https://github.com/python-poetry/poetry/issues/2310)) - one option might be [this plugin](https://github.com/nat-n/poethepoet) but given that you've brought in NPM, I know we can solve this there!

For example, we could:
* Install the `concurrently` package, so we can run two commands in parallel with a single command
* Add a scripts section to our `package.json` that looks something like:
```json
  "scripts": {
    "css": "npx tailwindcss -i ./todo_app/tailwind.css -o ./todo_app/static/css/index.css --watch",
    "dev": "concurrently \"npm run css\" \"poetry run flask run\""
  }
```
And then we can start both css and app together with `npm run dev`

Reply:
Ooooo thats very nice, I also didn't like having to run extra commands but hadn't considered enforcing it on run... but that makes a lot of sense as you would always want the current css 😄